kmath-trajectory/README.md

```diff
@@ -0,0 +1,32 @@
+# Module kmath-trajectory
```

Comment:
Be sure to add your contacts to `## Author` section at `module/docs/README-TEMPLATE.md`

Reply:
Added Author details, though seems out of place